.elasticbeanstalk/config.yml

```diff
@@ -1,17 +0,0 @@
-branch-defaults:
```

Comment:
why have you added this to the gitignore? its needed by travis in order to deploy